app/controllers/rx_controller.rb

```diff
@@ -4,13 +4,21 @@
 class RxController < ApplicationController
   include ActionController::Serialization
 
-  skip_before_action :authenticate
+  before_action :authorize_rx
   before_action :authenticate_client
 
   protected
 
   def client
-    @client ||= Rx::Client.new(session: { user_id: ENV['MHV_USER_ID'] })
+    @client ||= Rx::Client.new(session: { user_id: mhv_correlation_id })
+  end
+
+  def authorize_rx
+    mhv_correlation_id || raise(Common::Exceptions::Unauthorized)
```

Comment:
I've disabled forbidden. Forbidden is usually for a resource access permission, trying to maybe request a different users resources. I think unauthorized makes sense and anywhere we would traditionally use forbidden we instead do 404 not found. Forbidden is no longer considered best practice really because it implies a resource exists